pkg/middleware/cache.go

```diff
@@ -1,15 +1,17 @@
 package middleware
 
 import (
+	"errors"
 	"fmt"
 	"net/http"
 	"time"
 
 	"github.com/mikestefanello/pagoda/pkg/context"
 	"github.com/mikestefanello/pagoda/pkg/services"
 
-	"github.com/go-redis/redis/v8"
+	lib_store "github.com/eko/gocache/lib/v4/store"
```

Comment:
Can I ask that you remove the underscores in the import aliases?

Reply:
Sure, I don't particularly like it myself but I was following the convention that [gocache uses](https://github.com/eko/gocache/blob/96fc2f790c4cfb27659863657625fa8c4e68d25d/store/redis/redis.go#L8).